.github/workflows/verify.yml

```diff
@@ -44,6 +44,21 @@ jobs:
         run: npm i
       - name: Run linter
         run: npm run lint
+      - name: Detect changed documentation files
+        uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          list-files: shell
+          filters: |
+            docs:
+              - '*.md'
+              - '.github/**.md'
+      - name: Check documentation links
+        if: steps.changes.outputs.docs == 'true'
+        run: |
+          npx markdown-link-check --retry \
```

Comment:
This will not be cached and will always install the latest version.

We can consider adding this package to `devDependencies`.